package.json

```
@@ -1,25 +1,28 @@
 {
   "name": "eslint-config-mimic",
-  "version": "0.0.2",
+  "version": "0.0.3",
```

Let's do the bump with the new version tag on a separate commit
```suggestion
  "version": "0.0.2",
```